armi/operators/operator.py

```diff
@@ -376,10 +385,16 @@ def _timeNodeLoop(self, cycle, timeNode):
         self.r.p.timeNode = timeNode
         self.interactAllEveryNode(cycle, timeNode)
         # perform tight coupling if requested
-        if self.cs["numCoupledIterations"]:
-            for coupledIteration in range(self.cs["numCoupledIterations"]):
+        if self.cs["tightCoupling"]:
```

Comment:
So, if people use tight coupling, they will get all these print-outs at `info` level?

Just spit-balling here.  Are these print-outs something people would definitely want to see every single time they run Nala?  Or should these be `debug` level?  (Honest question.)

Reply:
In my opinion, yes, it's something I would always want to know. 

Since each nonlinear iteration can take some time to complete, I think it is best to print the tight coupling report summary table on each iteration. Commit https://github.com/terrapower/armi/pull/1033/commits/79809517dc7b502a2c63e0e46bdfbc4108d20184 cleans out an unnecessary/extra call to the print routine.